client/src/components/profile.ts

```diff
@@ -426,9 +426,6 @@ export class ProfileConfig {
         ProfilePromptType.ClientId,
         profileClone.clientId
       );
-      if (profileClone.clientId === "") {
```

Comment:
At the moment, we prevent writing config details that are the "default" case (in this case, an empty string). We do something similar for compute context (we only write it to a profile _if_ it's different than the default). 

For now, I wouldn't include this change. Some additional discussion is needed to know whether we should:
 - only write non-default values to a config
 - write all values to a config

In any case, what we do should be consistent for all properties with a default value.